server/chat-plugins/helptickets.ts

```diff
@@ -1210,24 +1255,18 @@ export const commands: ChatCommands = {
 		close(target, room, user) {
 			if (!target) return this.parse(`/help helpticket close`);
 			let result = !(this.splitTarget(target) === 'false');
```

Comment:
Why isn't CI complaining about an unused variable? I don't see this used anywhere.